### ISSUE TITLE
Auto Verify Not Working

### ISSUE THREAD

**Issue by DarkStarsGaming** [thejoabo/virtualfisher-bot#27]
Hello, I have noticed that the auto verify is not working, I am using a api key in my config but it is still not working for some reason.

This has happened multiple times and I am not sure of why this is happening, if it is possible to fix this please do because I got a 7 day ban because the auto verify wasn't working,
![image](https://user-images.githubusercontent.com/59534560/183253751-300fa073-b8b4-4c61-868c-2e7271d4c410.png)

**Comment by thejoabo** [comment]
Hello @DarkStarsGaming , as stated on the readme "_Keep in mind that the captcha detection method is not flawless, unexpected events can cause some unusual behavior that could influence detection accuracy. **Therefore, it should not be left alone without monitoring for longer periods of time.**_". 

I did several hours of testing with this new version (1.2.0), and never failed a detect even a single time (not always solved, but every time correctly detected), but yet, there is no possible way that I can predict every behavior that can occur (even though I use a testing method for the whole event - for instance, even if you change your name to "captcha" or "verify" the bot will crash and warn about an unknown event, because I intentionally making the method redundant to avoid detection violations).

The main problem is: how can I troubleshoot or reproduce your problem based on a screenshot of a Discord chat ? I cannot figure out what triggers the event from not being detected (and to be honest, nothing at all). So, if possible, help me to identify what's going on.

If you have any screenshot, dumps about the logs after the captcha not being detected, please send it here, otherwise:
- Are you using the autofishbot on a private channel ? (this means, no multiple people sending commands at the same time)
- Which version of the bot are you using ?
- During your time of use, the bot **never** solved or even detected a captcha?

Unfortunately, without further information or the logs for that specific moment, I can't help you. I'm sorry about the inconvenience.

**Comment by DarkStarsGaming** [comment]
Hello,

I was using the latest version of the bot, and it didn't seem like it ever did solve any captchas, when I checked back on my pc I would have to do many of them by hand.

But I was also doing it in a private channel.

**Comment by thejoabo** [comment]
Hello @DarkStarsGaming, yesterday, I cloned the current version of the bot and used it non-stop for about 5 hours straight while I was doing some other stuff, hoping that I could ran into your problem or maybe spot some weird behavior related to the captcha. Other than a sudden disconnection after ~2h, everything worked fine. I got a total of 9 captchas during this time, and all of them were automatically handled and solved.

Based on that, my only solution, at this time, for **your** specific case is to stop using this bot for its captcha purposes. 

 **I can't fix an error that I can't reproduce not even once.**
